cmplibrary/src/main/java/com/sourcepoint/cmplibrary/util/ViewsManager.kt

```diff
@@ -132,7 +128,6 @@ private class ViewsManagerImpl(
             check {
                 ConsentWebView(
                     context = it,
-                    connectionManager = connectionManager,
```

Comment:
no need this change

Reply:
We cannot delete the ConnectionManager, it is the only way to inject a mock and create UI tests for this feature